build_scripts/appimage/install-infection-monkey-service.sh

```diff
@@ -69,6 +77,10 @@ EOF
 }
 
 uninstall_service() {
+  if [ -d "${MONKEY_DATA_DIR}" ] ; then
+    sudo rm -rf "${MONKEY_DATA_DIR}"
```

Comment:
Running `sudo rm -rf` is super dangerous. There could be symlinks in this directory and who knows what could be removed.

We can run sudo with `-u <USER>` to limit the impact.